src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,9 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+
+    for(let i = 0; i < employees.length; i++)
+    if (employeeName === employees[i].name)
+    return employees[i].role;
+
```

Comment:
Nice solution. Extra challenge - do you think you can solve it using an iterator method like `.find()` instead?